app/components/Blurb.js

```diff
@@ -1,19 +1,29 @@
 import React from "react";
-import Image from 'next/image'
-import Link from 'next/link'
+import Image from "next/image";
+import Link from "next/link";
 
 export default function Blurb(props) {
-    return (
```

Comment:
please revert back to how the blurb was, i'm not sure you look at the generaluse of the blurb before adjusting it. you can see how it looks on the website now.